### PR TITLE
Feature 2

### PR DESCRIPTION
**Pull request by mahoward99**

### CODE REVIEW

**Comment by kacoates** [comment]
approved

**Comment by kacoates** [comment]
you are doing great sweety!

**Comment by Jjguzman1** [comment]
Approved homie!

File: PersonalTwitterFeed/src/personaltwitterfeed/PersonalTwitterFeed.java [hunk 1]
```diff
@@ -28,30 +30,48 @@ public static void main(String[] args) {
...
         
+        
         System.out.println("Nice to meet you " + tweeterName + "!");
         System.out.println("Enter your tweets and I will add them to your timeline!");
         
-        int numTweets = 0;
+       
         
         while(numTweets < (MAX_NUMBER_TWEETS - 1)) {
-            tweets[numTweets] = keyboard.nextLine();
```
Comment (kacoates) [inline PersonalTwitterFeed/src/personaltwitterfeed/PersonalTwitterFeed.java:37]: your great!

Comment (kacoates) [reply PersonalTwitterFeed/src/personaltwitterfeed/PersonalTwitterFeed.java:37]: almighty coder you!

**Review by ccannon94 — APPROVED** [review]
:100:

**Pull request merged**